packages/treat/tests/__snapshots__/styling.test.ts.snap

```diff
@@ -159,6 +159,24 @@ Object {
 exports[`Styling and specificity Themed - development mode (style-loader) should be styled correctly with theme one 1`] = `
 Object {
   "background-color": "yellow",
+  "border": "6px solid pink",
+  "border-bottom-color": "pink",
+  "border-bottom-style": "solid",
+  "border-bottom-width": "6px",
+  "border-image-outset": "initial",
+  "border-image-repeat": "initial",
+  "border-image-slice": "initial",
+  "border-image-source": "initial",
+  "border-image-width": "initial",
+  "border-left-color": "pink",
+  "border-left-style": "solid",
+  "border-left-width": "6px",
+  "border-right-color": "pink",
+  "border-right-style": "solid",
+  "border-right-width": "6px",
+  "border-top-color": "pink",
+  "border-top-style": "solid",
+  "border-top-width": "6px",
```

Comment:
Shorthand properties tend to be pretty noisy in the snapshots so I've avoided until now. Thoughts?

Reply:
Yep, let's clean this up 👍